__tests__/bin-watch-serve.js

```diff
@@ -63,9 +63,9 @@ test(
     return pEvent(docProcess.stdout, 'data').then(function(data) {
       const portNumber = data
         .toString()
-        .match(/documentation.js serving on port (\d+)/);
+        .match(/documentation.js serving on port 4004/);
```

Comment:
This test was passing regardless of which port documentation.js was being served on.